www/index.html

```diff
@@ -51,6 +51,53 @@ <h1>Locomotive Boilerplate</h1>
             <main data-scroll-section>
                 <div class="o-container">
                     <h1 class="c-heading -h1">Hello</h1>
+
+                    <!--
+                    --   ICONS
+                    -->
+
+                    <!-- Icon with accessible label -->
+                    <i class="o-icon">
+                        <svg
+                            class="svg-logo"
+                            xmlns="http://www.w3.org/2000/svg"
+                            role="img"
+                            aria-labelledby="icon-logo-label"
+                        >
+                            <title id="icon-logo-label">
+                                Locomotive
+                            </title>
+                            <use xlink:href="assets/images/sprite.svg#logo" xmlns:xlink="http://www.w3.org/1999/xlink"/>
+                        </svg>
+                    </i>
+
+                    <!-- Icon with custom colors -->
+                    <i class="o-icon">
+                        <svg
+                            class="svg-logo -invert"
+                            xmlns="http://www.w3.org/2000/svg"
+                            role="img"
+                        >
+                            <use xlink:href="assets/images/sprite.svg#logo" xmlns:xlink="http://www.w3.org/1999/xlink"/>
+                        </svg>
+                    </i>
+
+                    <!-- Icon with size -->
+                    <i class="o-icon -big">
+                        <svg
+                            class="svg-logo"
+                            xmlns="http://www.w3.org/2000/svg"
+                            role="img"
+                        >
+                            <use xlink:href="assets/images/sprite.svg#logo" xmlns:xlink="http://www.w3.org/1999/xlink"/>
+                        </svg>
+                    </i>
```

Comment:
If these images are meant to be decorative / have no accessible name, they should use `aria-hidden="true" focusable="false"` instead of `role="img"`.

Reply:
That's true.
It was some usage examples that are going ot be deleted if merge.
I can add the markup in comments in the `_icons.scss` file with the specifications